pyproject.toml

```diff
@@ -53,6 +53,62 @@ dependencies = [
 ]
 
 [project.optional-dependencies]
+alignment = [
+    "dtw-python",
```

Comment:
should also have `tslearn`, no?

Reply:
I never used alignment, but VS Code search couldn't find that against `python_dependencies` tag. Can you please point out the relevant line/module/sub-package?